Hmm, that's new:
```
"D:\a\Tooling-Windows-Submodule\Tooling-Windows-Submodule\components\CiTestExp\CiTestExp.sln" (default target) (1:2) ->
"D:\a\Tooling-Windows-Submodule\Tooling-Windows-Submodule\components\CiTestExp\heads\Tests.Wasdk\CiTestExp.Tests.Wasdk.csproj" (default target) (2:6) ->
(MsixCheckForInvalidCustomBeforeMicrosoftCommonTargets target) -> 
  C:\Users\runneradmin\.nuget\packages\microsoft.windows.sdk.buildtools.msix\1.7.20250829.1\build\Microsoft.Windows.SDK.BuildTools.MSIX.targets(33,5): error : The MSIX build tools use the 'CustomBeforeMicrosoftCommonTargets' MSBuild property to wire up a custom .props file that sets some necessary properties. This .props file does not appear to have been imported correctly. This likely means that someone has also overridden 'CustomBeforeMicrosoftCommonTargets', without ensuring to also chain the import of the previously assigned .props and .targets set to that property. You can use a binlog to learn more about where that assignment was done (set the verbosity to diagnostics, and search for 'reassignment: $(CustomBeforeMicrosoftCommonTargets'). [D:\a\Tooling-Windows-Submodule\Tooling-Windows-Submodule\components\CiTestExp\heads\Tests.Wasdk\CiTestExp.Tests.Wasdk.csproj]


"D:\a\Tooling-Windows-Submodule\Tooling-Windows-Submodule\components\CiTestExp\CiTestExp.sln" (default target) (1:2) ->
"D:\a\Tooling-Windows-Submodule\Tooling-Windows-Submodule\components\CiTestExp\heads\Wasdk\CiTestExp.Wasdk.csproj" (default target) (3:6) ->
(_ValidatePresenceOfAppxManifestItems target) -> 
  C:\Users\runneradmin\.nuget\packages\microsoft.windows.sdk.buildtools.msix\1.7.20250829.1\build\Microsoft.Windows.SDK.BuildTools.MSIX.Packaging.targets(3024,5): error MSB4062: The "Microsoft.Build.Msix.AppxManifest.WinAppSdkValidateAppxManifestItems" task could not be loaded from the assembly C:\Users\runneradmin\.nuget\packages\microsoft.windowsappsdk\1.6.250108002\buildTransitive\..\tools\net472\Microsoft.Windows.SDK.BuildTools.MSIX.dll. Could not load file or assembly 'file:///C:\Users\runneradmin\.nuget\packages\microsoft.windowsappsdk\1.6.250108002\tools\net472\Microsoft.Windows.SDK.BuildTools.MSIX.dll' or one of its dependencies. The system cannot find the file specified. Confirm that the <UsingTask> declaration is correct, that the assembly and all its dependencies are available, and that the task contains a public class that implements Microsoft.Build.Framework.ITask. [D:\a\Tooling-Windows-Submodule\Tooling-Windows-Submodule\components\CiTestExp\heads\Wasdk\CiTestExp.Wasdk.csproj]
```

Errors from the latest CI run.

I notice it's coming from `C:\Users\runneradmin\.nuget\packages\microsoft.windowsappsdk\1.6.250108002\buildTransitive\..\tools\net472\Microsoft.Windows.SDK.BuildTools.MSIX.dll.`, so there might still be a version number to update somewhere.

We don't depend on Win2D here, but it might be coming from Behaviors.

--- a/ProjectHeads/App.Head.WinAppSdk.Dependencies.props
+++ b/ProjectHeads/App.Head.WinAppSdk.Dependencies.props
@@ -3,6 +3,6 @@
 <Project>
   <ItemGroup>
     <PackageReference Include="CommunityToolkit.WinUI.UI.Controls.Markdown" Version="7.1.2" />
-    <PackageReference Include="Microsoft.Xaml.Behaviors.WinUI.Managed" Version="3.0.0" />
+    <PackageReference Include="Microsoft.Xaml.Behaviors.WinUI.Managed" Version="3.0.1" />
   </ItemGroup>
 </Project>

